keys.json

```diff
@@ -45,6 +45,7 @@
     "coolcoin":      { "skip": true },
     "crypton":       { "skip": true },
     "dsx":           { "skip": true },
+    "duedex":        { "skip": true },
```

Comment:
Hi @paulran ! Can you plz remove this edit from keys.json?

Reply:
Because duedex doesn't  support fetchOrderBook, if not skip,  running bash -c 'node run-tests --js --python 5' will fail.